kayvee.go

```diff
@@ -19,7 +20,11 @@ const (
 
 // Format converts a map to a string of space-delimited key=val pairs
 func Format(data map[string]interface{}) string {
-	formattedString, _ := json.Marshal(data)
+	formattedString, err := json.Marshal(data)
+	if err != nil {
+		log.Printf("Error converting kayvee message to json: %s", err.Error())
```

Comment:
Maybe this should be logged in kayvee format?
![](http://fc02.deviantart.net/fs70/f/2012/220/5/e/xzibit_yo_dawg_render_by_kernelpanicx-d5aa710.png)